Solutions/ARGOSCloudSecurity/Analytic Rules/ExploitableSecurityIssues.yaml

```diff
@@ -0,0 +1,45 @@
+id: a9bf1b8c-c761-4840-b9a8-7535ca68ca28
+name: ARGOS Cloud Security - Exploitable Cloud Resources
+description: Exploitable Cloud Security Issues are ones that expose cloud resources   to the internet and allow initial access to your environment.
+requiredDataConnectors:
+  - connectorId: ARGOSCloudSecurity
+    dataTypes:
+      - ARGOS_CL
+kind: Scheduled
+version: 1.0.0
+severity: High
+query: |
+  ARGOS_CL | where exploitable_b
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+suppressionEnabled: false
+tactics:
+- InitialAccess
+techniques: []
+alertRuleTemplateName:
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: false
+    reopenClosedIncident: false
+    lookbackDuration: 5h
+    matchingMethod: AllEntities
+    groupByEntities: []
+    groupByAlertDetails: []
+    groupByCustomDetails: []
+eventGroupingSettings:
+  aggregationKind: AlertPerResult
+alertDetailsOverride:
+customDetails:
+entityMappings:
```

Comment:
I see there is an userId in the auditlog_s field and there is a Computer field in the sample data, any reason we are not bringing those thru as entities here?

Reply:
The `userId` in the auditlog is currently only used for actions taken on the ARGOS dashboard. I don't think it'd add value here.
Currently investigating the `Computer` field as this is not something we typically use.

Reply:
@davidobrien1985 - Sounds good, thanks for clarifying.